yjit.rb

```diff
@@ -184,6 +184,16 @@ def self.runtime_stats(context: false)
     stats
   end
 
+  # Format and print out counters as a String. This returns a non-empty
+  # content only when --yjit-stats is enabled.
+  def self.printed_stats
```

Comment:
```suggestion
  def self.stats_string
```

Seems like a good idea. Maybe this would be a more descriptive name?

Reply:
:+1: Renamed https://github.com/ruby/ruby/pull/7857/commits/f3705b63d206eba9fedea05ca378dc0da78e7f86